packages/transaction-controller/src/TransactionController.ts

```diff
@@ -819,7 +840,7 @@ export class TransactionController extends BaseController<
       originalGasEstimate: transactionMeta.txParams.gas,
       type: TransactionType.retry,
     };
-    const newTransactionMeta =
+    const updatedTransactionMeta =
```

Comment:
Technically, is this not new transaction metadata as we're creating a new transaction entry in the state?

Reply:
True, I also couldn't understand why I made this change. Thanks for pointing out, fixed in https://github.com/MetaMask/core/pull/1894/commits/951182dd5b77ecaad6ec699a22b86a1fc9193931